client/v3/naming/endpoints/internal/update.go

```diff
@@ -0,0 +1,37 @@
+package internal
```

Comment:
Where do we use this package?

Reply:
Inside implementation of the endpoint.Manager. It describes how we serialize and deserialize the endpoints. 

As client/v3 is being is linked into many customer's application, it's important to distinguish public API surface from implementation details. We could keep it lower-case as well, but I wanted in this PR stress its private.

Reply:
I understand that, but `client/v3/naming/endpoints/endpoints_impl.go` importing this looks like placeholders and not implemented yet?

Reply:
Yes. I hope for contributors to implement this within a month. If no one will contribute, I will do the implementation itself. 
The endpoints definition is fixed in stone so its why I'm submitting it.